webui/ocw/lib/vault.py

```diff
@@ -90,6 +101,35 @@ def renew(self):
         self.revoke()
         self.getData()
 
+    def getAuthCacheFile(self):
```

Comment:
I think this name would be more accurate 
```suggestion
    def createAuthCachePath(self):
```